src/pages/templatetags/library.py

```diff
@@ -42,3 +42,7 @@ def highlight(text, search):
 @register.filter(is_safe=True)
 def period_names(assignment_list):
     return ", ".join([p.period.name for p in assignment_list])
+
+@register.filter(is_safe=True)
+def starred_val(paradata_set):
+    return ", ".join([str(p.starred).lower() for p in paradata_set])
```

Comment:
I think we can avoid the need for this unbeautiful thing (though learning how to create filters is valuable). Let me play around with it a bit so that I don't suggest something that doesn't work.